src/info/langs/mod.rs

```diff
@@ -22,7 +22,12 @@ pub fn get_loc_by_language_sorted(
     let locs = get_locs(dir, globs_to_exclude, language_types, include_hidden);
 
     let loc_by_language =
-        get_loc_by_language(&locs).context("Could not find any source code in this repository")?;
+        get_loc_by_language(&locs).with_context(|| {
+            format!(
+                "Could not find any source code in this repository at path '{}'. Try using --include-hidden if your files are hidden, or check your exclude patterns.", 
```

Comment:
The most common point of confusion I've seen is languages that are excluded by default not being shown. For example, JSON, a data language, is excluded by default, so a repo that is only JSON files would have "no source code" unless they use the `-T` option. Perhaps you should mention that instead.